include/tesseract/baseapi.h

```diff
@@ -559,6 +559,18 @@ class TESS_API TessBaseAPI {
    */
   char *GetAltoText(int page_number);
 
+   /**
+   * Make an XML-formatted string with PAGE markup from the internal
+   * data structures.
+   */
+  char *GetPAGEText(ETEXT_DESC *monitor, int page_number);
+
+  /**
+   * Make an XML-formatted string with PAGE markup from the internal
+   * data structures.
+   */
+  char *GetPAGEText(int page_number);
+
```

Comment:
Would it be better to combine the two functions in a single function `char *GetPAGEText(int page_number, ETEXT_DESC *monitor = nullptr)`?

Reply:
For now, it's consistent with all the other renders. See lines above.

Reply:
I know, but we are free to improve things if that's possible. I never likes the old API very much. Adding lots of new functions for each new renderer is not very elegant. Maybe someone has a better idea.